tests/test_cli.py

```diff
@@ -107,3 +108,79 @@ def test_create_user_config():
         "foo": None,
         "main": 3,
     }
+
+
+class MockResponse:  # pylint: disable=too-few-public-methods
+    def __init__(self, data, status_code):
+        self.data = data
+        self.status_code = status_code
+
+    def json(self):
+        return self.data
+
+
+@pytest.fixture(name="authenticated_cli")
+def fixture_authenticated_cli():
+    def patched_get_auth_token():
+        return "token"
+
+    cli = AivenCLI()
+    cli._get_auth_token = patched_get_auth_token  # pylint: disable=protected-access
+    return cli
+
+
+def test_service_task_create_migration_check(authenticated_cli):
+    task_create_response = {
+        "message": "created",
+        "task": {
+            "create_time": "2021-08-13T08:00:45Z",
+            "result": "",
+            "success": None,
+            "task_id": "79803598-d09a-44bf-ae2b-34aad942f4e8",
+            "task_type": "mysql_migration_check"
+        }
+    }
+
+    with mock.patch("aiven.client.client.AivenClient") as cli_client_mock:
+        cli_client_mock.return_value.create_service_task.return_value = MockResponse(task_create_response, 200).json()
```

Comment:
This is equivalent to `... = task_create_response`, isn't it?

(same for below)

Reply:
I should have polished it :) Yeah, not mocking class not needed. Thanks!